package.json

```diff
@@ -27,12 +27,14 @@
     "test:ember": "ember test"
   },
   "devDependencies": {
+    "@appuniversum/appuniversum": "^1.2.0",
     "@appuniversum/ember-appuniversum": "^1.1.1",
     "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "^2.6.0",
     "@glimmer/component": "^1.0.4",
     "@glimmer/tracking": "^1.0.4",
-    "@lblod/ember-acmidm-login": "^1.4.0",
+    "@lblod/ember-acmidm-login": "^1.4.0",    
+    "@lblod/ember-environment-banner": "git://github.com/lblod/ember-environment-banner.git#env-banner-op",
```

Comment:
Here usually we don't commit that, we wait until the package is released with our changes and we bump it